<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employguiClass</class>
 <widget class="QWidget" name="employguiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employ</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>231</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="employeeList"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addEmployeeButton">
          <property name="text">
           <string>Add Employee</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addSaleButton">
          <property name="text">
           <string>Add Sale</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="removeEmployeeButton">
        <property name="text">
         <string>Remove Employee</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="saleLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="totalLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="employgui.qrc"/>
 </resources>
 <connections/>
</ui>
